src/index.js

```diff
@@ -3,10 +3,12 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 
+import AddItemPage from "./components/AddItemPage";
+
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <AddItemPage />
```

Comment:
This can be set back to App after this week, once our demo is done and we start adding in the routing.